shell/common/common_benchmarks.sh

```diff
@@ -965,37 +965,47 @@ restart_monit(){
 run_monit() {
   local perf_mon="$1"
   local perf_mon_bin="$(get_local_bench_path)/aplic/${perf_mon}_$PORT_PREFIX"
+  local pids=()
 
   if [ "$perf_mon" == "sar" ] ; then
     if [ "$clusterType" != "PaaS" ]; then
-      $DSH "$(get_user_bin_path) $perf_mon_bench_path/${perf_mon}_$PORT_PREFIX -o $(get_local_bench_path)/sar-\$(hostname).sar $BENCH_PERF_INTERVAL >/dev/null &" & #2>&1
+      $DSH "$(get_user_bin_path) $perf_mon_bench_path/${perf_mon}_$PORT_PREFIX -o $(get_local_bench_path)/sar-\$(hostname).sar $BENCH_PERF_INTERVAL >/dev/null &" &
```

Comment:
Since a few months ago we are trying to avoid using $DSH to execute a command in all cluster machines. Instead use execute_all "string to define what are you doing (for logging)" "command to execute in all machines"

Reply:
OK, but my change only removed the commented redirection at the end.

Reply:
Just looking at `execute_all`, looks like a wrapper around a wrapper around a wrapper around a whole bunch of timing code which doesn't make much sense here since we're just starting the monitor processes. I'm going to leave it as it is for now I think.

Reply:
Problem is, we don't know if dsh is going to be installed in all the clusters we are running, specially PaaS or IaaS. That was the solution to that poblem, and it works without using dsh.

Reply:
Aloja used to assume that dsh was installed, either through the distro's package manager or built from source (the deployment script checks for that and builds a version from source if it's not available and cannot be installed as distro package)

Reply:
On those systems I guess we're not able to run this monitoring code anyway, right? If `execute_all` only abstracted around the `dsh` command, I'd be happy to go ahead and change it, but it introduces a whole other load of complications which only seem relevant to executions of benchmarks, not starting performance monitors.